lib/twitch-eventsub-ws/tests/src/string.cpp

```diff
@@ -567,3 +569,18 @@ TEST(String, Equals)
     ASSERT_TRUE(reallyLong == String(REALLY_LONG));
     ASSERT_TRUE(reallyLongQt == String(REALLY_LONG));
 }
+
+TEST(String, QStringBuilder)
+{
+    String s(MINI);
+
+    ASSERT_EQ("foo" % s.qt() % "bar", "foominibar");
+    ASSERT_EQ(u"foo" % s.qt() % "bar", "foominibar");
+    ASSERT_EQ("foo" % s.qt() % u"bar", "foominibar");
+    ASSERT_EQ(u"foo" % s.qt() % u"bar", "foominibar");
+
+    // the code below breaks on qt 6.4.3, but i think it's unrelated to our String stuff
+    // auto text2 = u"a" % s.qt() % u"b";
```

Comment:
A trivial fix would be 
```suggestion
    QString text2 = u"a" % s.qt() % u"b";
```